docs/contributing.md

```diff
@@ -25,6 +25,24 @@ Guidelines](https://opensource.google/conduct/).
 
 ## Contribution process
 
+### Code Style
+
+We use the automatice code formatter [Pyink](https://github.com/google/pyink), which is a fork of the [Black](https://github.com/psf/black) formatter with a few different behaviors tailored towards Google internal repository.
+
+#### Step 1: Install Pyink.
+
+*For Googlers:*
+
+Run `sudo apt install pipx; pipx install pyink` on your Cloudtop to install Pyink. 
```

Comment:
Is `pipx` required?

Reply:
Yes, pipx is required, so we could work around to install 3rd party packages on Cloudtop.